.gitignore

```diff
@@ -125,4 +125,4 @@ dmypy.json
 
 .idea/
 
-events.json
\ No newline at end of file
+*.json
```

Comment:
super nit: I don't think we should exclude all json files, maybe store this particular json files into a separated folder and ignore that? or ignore these files specifically?

Reply:
I think for the purpose of this sample script, this is fine.